thavalon-webapp/src/utils/AccountManager.tsx

```diff
@@ -25,6 +25,19 @@ interface JwtType {
     "expires_at": number,
 };
 
+interface CreateGameResponse {
+    "friendCode": string,
+}
+
+interface JoinGameInfo {
+    "friendCode": string,
+    "displayName": string,
+}
+
+interface JoinGameResponse {
+    "socketUrl": string,
```

Comment:
I didn't know this worked and I guess it's style, but you don't need quotes around the object keys. Up to you though if you want to keep or remove them.

Reply:
looks like most interfaces don't have quotes will remove... everywhere